### PR TITLE
Allow deletion of specific projects' analysis results

### PR DESCRIPTION
**Pull request by mariogiov**
Includes bugfixes for the Charon status update issues

### CODE REVIEW

**Comment by vezzi** [comment]
Looks fine with me, @Galithil can you review also and Merge if you are satisfied?

File: ngi_pipeline/engines/piper_ngi/local_process_tracking.py [hunk 1]
```diff
@@ -406,7 +406,8 @@ def record_process_sample(project, sample, workflow_subtask, analysis_module_nam
                                                                        e.message))
         extra_args = None
         if workflow_subtask == "merge_process_variantcall":
-            status_field = "alignment_status"
+            seqrun_status_field = "alignment_status"
+            sample_status_field = "UNDER_ANALYSIS"
```
Comment (Galithil) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: I'm pretty sure there is no field named "UNDER_ANALYSIS" in charon

**Comment by Galithil** [comment]
I'll merge that and PR the required changes.

File: ngi_pipeline/engines/piper_ngi/local_process_tracking.py (hunk 1 above)
Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: Ha dammit. Why do I code these so late at night.

I mean, I'm testing the PR reviewers.

**Pull request merged**